reconcile/slack_base.py

```diff
@@ -84,10 +83,29 @@ def slackapi_from_permissions(
 
     api = SlackApi(
         workspace_name,
-        token,
-        secret_reader,
+        token=secret_reader.read(token),
         init_usergroups=init_usergroups,
         api_config=api_config,
     )
 
     return api
+
+
+def get_slackapi(
```

Comment:
Is this method for testing only? Can't find any reference? :confused:

Reply:
See PR description. This method will be used later in [APPSRE-6592 - refactor slack-usergroups](https://issues.redhat.com/browse/APPSRE-6592). Basically, this supports getting a `SlackAPI` via GQL settings